### DISCUSSION TITLE
Peerjs server on IIS?

### DISCUSSION THREAD

**Discussion by spj-uk** [peers/peerjs#1216] [Q&A] [answered]
Is it possible to deploy peerjs-server on to a Windows Server?

**Comment by jonasgloning** [comment, marked as answer]
Sure! Node and PeerServer run on Windows Server.
If you want to use IIS, you’ll have to create a reverse proxy back to Node.

Also, If you're at home in the Windows/.NET-world, checkout [peers-server.net](https://github.com/cristipufu/peerjs-server.net), it’s a rewrite of this project in .NET that probably can be deployed to IIS directly.

**Comment by spj-uk** [comment]
Hi,

Thanks, does it still require a reverse proxy?

Simon.
